include/value.h

```diff
@@ -1197,8 +1197,8 @@ class PHPCPP_EXPORT Value : private HashParent
     /**
      *  Functions that need access to the privates
      */
-    friend Value constant(const char *name, size_t size);
-    friend bool  define(const char *name, size_t size, const Value &value);
+    friend PHPCPP_EXPORT Value constant(const char *name, size_t size);
+    friend PHPCPP_EXPORT bool  define(const char *name, size_t size, const Value &value);
```

Comment:
This and other items from the fise should be a separate PR. The issue will show up with any build with the strict visibility flags.

Thanks.